crates/cargo-test-support/src/paths.rs

```diff
@@ -287,3 +287,54 @@ pub fn sysroot() -> String {
     let sysroot = String::from_utf8(output.stdout).unwrap();
     sysroot.trim().to_string()
 }
+
+/// Returns true if names such as aux.* are allowed.
+///
+/// Traditionally, Windows did not allow a set of file names (see `is_windows_reserved`
+/// for a list). More recent versions of Windows have relaxed this restriction. This test
+/// determines whether we are running in a mode that allows Windows reserved names.
+#[cfg(windows)]
+pub fn windows_reserved_names_are_allowed() -> bool {
```

Comment:
Can this add something like:

```rust
if is_ci() { return false; }
```

I'd like to ensure the tests are still run on CI until such time that we need to migrate.

Reply:
Sure, I just pushed the change that does this.